examples/11_unit_input_folder.py

```diff
@@ -0,0 +1,168 @@
+__license__   = 'MIT'
+
+import os
+import sys
+
+verbose  = os.environ.get('RADICAL_PILOT_VERBOSE', 'REPORT')
+os.environ['RADICAL_PILOT_VERBOSE'] = verbose
+
+import radical.pilot as rp
+import radical.utils as ru
+
+
+# ------------------------------------------------------------------------------
+#
+# READ the RADICAL-Pilot documentation: http://radicalpilot.readthedocs.org/
+#
+# ------------------------------------------------------------------------------
+
+
+#------------------------------------------------------------------------------
+#
+if __name__ == '__main__':
+
+    # we use a reporter class for nicer output
+    report = ru.LogReporter(name='radical.pilot', level=verbose)
+    report.title('Getting Started (RP version %s)' % rp.version)
+
+    # use the resource specified as argument, fall back to localhost
+    if   len(sys.argv)  > 2: report.exit('Usage:\t%s [resource]\n\n' % sys.argv[0])
+    elif len(sys.argv) == 2: resource = sys.argv[1]
+    else                   : resource = 'local.localhost'
+
+    # Create a new session. No need to try/except this: if session creation
+    # fails, there is not much we can do anyways...
+    session = rp.Session()
+
+    # all other pilot code is now tried/excepted.  If an exception is caught, we
+    # can rely on the session object to exist and be valid, and we can thus tear
+    # the whole RP stack down via a 'session.close()' call in the 'finally'
+    # clause...
+    try:
+
+        # read the config used for resource details
+        report.info('read config')
+        config = ru.read_json('%s/config.json' % os.path.dirname(os.path.abspath(__file__)))
+        report.ok('>>ok\n')
+
+        report.header('submit pilots')
+
+        # Add a Pilot Manager. Pilot managers manage one or more ComputePilots.
+        pmgr = rp.PilotManager(session=session)
+
+        # Define an [n]-core local pilot that runs for [x] minutes
+        # Here we use a dict to initialize the description object
+        pd_init = {
+                'resource'      : resource,
+                'runtime'       : 15,  # pilot runtime (min)
+                'exit_on_error' : True,
+                'project'       : config[resource]['project'],
+                'queue'         : config[resource]['queue'],
+                'access_schema' : config[resource]['schema'],
+                'cores'         : config[resource]['cores'],
+                }
+        pdesc = rp.ComputePilotDescription(pd_init)
+
+        # Launch the pilot.
+        pilot = pmgr.submit_pilots(pdesc)
+
+        ## create folder and a file that is going to be moved 
+        directory ='/tmp/stage_in_folder'
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+
+        # create a folder in the file
+        filename = '/tmp/stage_in_folder/input_file.dat'
+        afile = open(filename,'w')
+        afile.write('hello world!')
+        afile.close()
+
+        # TODO: transfer file to remote machine
```

Comment:
You mean transfer the folder? 

Also, how does this work currently? do you already have the folder in /tmp/ ?

Reply:
Yes, I only test in localhost right now, so I did not have to move the folder. 
After our discussion, I will create the CU who creates the directory.